Testing/functional/tests/cypress/integration/ui/pages/LandingPage.js

```diff
@@ -117,3 +117,31 @@ describe("Landing Page", () => {
         cy.get("footer > .navbar").should("not.visible");
     });
 });
+
+describe("Landing Page - Too Many Requests", () => {
+    beforeEach(() => {
+        cy.logout();
+    });
+
+    it("Too Many Requests Banner Appears on 429 Response", () => {
+        cy.intercept("GET", "**/Communication/*", { statusCode: 429 });
+        cy.visit("/");
+        cy.contains(
+            "[data-testid=communicationBanner]",
+            "higher than usual site traffic"
+        ).should("be.visible");
+    });
+
+    it("Too Many Requests Banner Doesn't Appear on 200 Response", () => {
+        cy.intercept("GET", "**/Communication/*", { statusCode: 200 }).as(
+            "getCommunication"
+        );
+        cy.visit("/");
+        cy.wait("@getCommunication");
```

Comment:
Looks like line 140 and 141 are the same.

Reply:
That's intentional - it's waiting for both of the Communications calls (for public and in-app banner) to complete before checking that the banner is not present.  I should add a comment to that effect, though.